lib/home_page.dart

```diff
@@ -33,21 +33,10 @@ class _HomePageState extends State<HomePage> {
                       padding: const EdgeInsets.symmetric(vertical: 8.0),
                       child: ElevatedButton(
                         onPressed: () {
-                          Navigator.of(context).push(MaterialPageRoute(
-                            builder: (context) {
-                              if (snapshot.hasData) {
-                                return DetailPage();
-                              }
-                              if (snapshot.connectionState ==
-                                  ConnectionState.waiting) {
-                                return CircularProgressIndicator();
-                              }
-                              if (snapshot.hasError) {
-                                return Text('Could not load page');
-                              }
-                              return Text('Error');
-                            },
-                          ));
+                          var fsqId = snapshot.data![index].fsq_id;
+                          print('DEBUG press');
```

Comment:
This is only been used to debug, you don't need it anymore. Therefore, delete this print.

Reply:
done